src/modules/roc_packet/fec.h

```diff
@@ -21,10 +21,18 @@ namespace packet {
 
 //! FECFRAME packet.
 struct FEC {
-    //! Seqnum of first source packet in block.
-    seqnum_t blknum;
+    //! Number of a source block in a packet stream.
+    //!
+    //! @remarks
+    //!  Source block is formed from the source packets.
+    //!  Blocks are numbered sequentially starting from a random number.
+    //!  Block number can wrap.
+    blknum_t source_block_number;
 
-    //! The number of source packet per block.
+    //! Number of source packets in the block to which this packet belongs to.
+    //!
+    //! @remarks
+    //!  Different blocks can have different number of source packets.
     size_t source_block_length;
 
     //! The index number of the repair packet in block.
```

Comment:
I think we also should rename "repair_symbol_id" to "encoding_symbol_id", as it is called in the RFC.

Reply:
Done